cmd/queue/main.go

```diff
@@ -89,6 +90,7 @@ func initEnv() {
 	servingRevision = util.GetRequiredEnvOrFatal("SERVING_REVISION", logger)
 	servingAutoscaler = util.GetRequiredEnvOrFatal("SERVING_AUTOSCALER", logger)
 	servingAutoscalerPort = util.GetRequiredEnvOrFatal("SERVING_AUTOSCALER_PORT", logger)
+	userTargetPort = util.GetRequiredEnvOrFatal("PORT", logger)
```

Comment:
"PORT" here may cause some confusion since it is not clear this env is the port of user container or other containers. I would use environment variable "USER_PORT" in queue proxy. For user container, we still use "PORT".

Reply:
Thanks , I will change the env name.